docs/en/best_practice.md

```diff
@@ -1,6 +1,68 @@
 #  Best practices
 
 ## 1 Producer
+### 1.1 Attention of send message

```

Comment:
I really appreciate your attention to help finish the english copies of the user manual. Would you like to make another pr to these parts? Let pr solve only one issue.

Reply:
ok， I‘ll create another issue